tools/AzureRM/AzureRM.psd1

```diff
@@ -48,7 +45,36 @@ CLRVersion='4.0'
 ProcessorArchitecture = 'None'  
   
 # Modules that must be imported into the global environment prior to importing this module  
-RequiredModules = @() 
+RequiredModules = @(
+    @{ ModuleName = 'AzureRM.Profile'; MaximumVersion = '1.9999.9999.9999'; ModuleVersion = '1.0.4'},
```

Comment:
Is there any script to update the Module Version dependencies in this module (was there any such script for updating the versions in the script)?